mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -261,6 +262,34 @@ export class CompareRunView extends Component {
         <tbody>{dataRows}</tbody>
       </table>
     );
+  }  
+
+  renderTagTable(colWidth) {
+    const dataRows = this.renderDataRows(
+      this.props.tagLists,
```

Comment:
@CaioCavalcanti We discussed whether we should show MLflow system tags today. Here's our conclusion:

> If we show system tags on the run comparison page, we should probably show them on all pages (e.g. Runs table, Run Page, etc) for consistency. However, the Run Page lets you edit tags, yet system tags should not be edited arbitrarily. This could create confusion and buggy behavior in features that depend on particular system tag values. For this reason, I think it's best not to surface system tags in the UI and instead encourage users to call the API if they're interested in viewing these tags.

Reply:
Function to filter out MLflow system tags:

https://github.com/mlflow/mlflow/blob/4c0f21025322f4681e5c4cf01d2ad57e0eb93115/mlflow/server/js/src/common/utils/Utils.js#L798-L805

Reply:
Done, now it shows only the visible tags.